src/Client.ts

```diff
@@ -134,18 +140,27 @@ export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
  */
 export default class Client {
   address: string
-  keys: PrivateKeyBundleV1
+  legacyKeys: PrivateKeyBundleV1
+  keys: PrivateKeyBundleV2
   apiClient: ApiClient
   private contacts: Set<string> // address which we have connected to
-  private knownPublicKeyBundles: Map<string, PublicKeyBundle> // addresses and key bundles that we have witnessed
+  private knownPublicKeyBundles: Map<
+    string,
+    PublicKeyBundle | SignedPublicKeyBundle
+  > // addresses and key bundles that we have witnessed
```

Comment:
peer keys now can be V1 or V2, this will be used as a signal to conduct v1 or v2 conversation.